.changeset/two-mails-boil.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/catalog-client': minor
+---
+
+**BREAKING**: Removed `CatalogClient.getLocationByEntity` and `CatalogClient.getOriginLocationByEntity` which had previously been deprecated. Please use `CatalogApi.getLocationByRef` instead.
```

Comment:
Could be worth pointing out that this only applies if the `CatalogClient` was used directly, as it was already removed from the API

Reply:
aight, added clarification